tests/test_storage.py

```diff
@@ -7,11 +7,12 @@
 import hiro
 import redis
 import redis.lock
+from redis.sentinel import Sentinel
 from limits.strategies import FixedWindowRateLimiter, MovingWindowRateLimiter
 from limits.errors import ConfigurationError
 from limits.limits import RateLimitItemPerMinute, RateLimitItemPerSecond
 from limits.storage import (
-    MemoryStorage, RedisStorage, MemcachedStorage,
+    MemoryStorage, RedisStorage, MemcachedStorage, RedisSentinelStorage,
     Storage, storage_from_string
 )
 
```

Comment:
Please add a `flushall` call in the `setUp` method for storage tests to help ensure a clean state for the test runs.